examples/llm/inference/benchmarking/yamls/7b.yaml

```diff
@@ -0,0 +1,45 @@
+benchmark_name: MosaicDeepspeed_1xA100-40GB-fp16-GPT-7B
+
+max_seq_len: 2048
+tokenizer_name: gpt2
+
+# Tokenizer
+tokenizer:
+  name: ${tokenizer_name}
+  kwargs:
+    model_max_length: ${max_seq_len}
+
+# Important
+
+model:
+  name: mosaic_gpt
+  init_device: cpu
+  tokenizer_name: ${tokenizer_name}
+  d_model: 4096
+  n_heads: 32
+  n_layers: 32
+  mlp_ratio: 4
+  max_seq_len: ${max_seq_len}
+  vocab_size: 50368
+  init_std: 0.02
+  attn_pdrop: 0.0
+  resid_pdrop: 0.0
+  emb_pdrop: 0.0
+  attn_impl: triton
+
+fsdp_config:
+  sharding_strategy: FULL_SHARD
+  mixed_precision: DEFAULT
+  activation_checkpointing: false
+  activation_cpu_offload: false
+  verbose: false
+
+load_model_path: MosaicGPT.pt
```

Comment:
Probably just leave this blank?

Reply:
I think for now I'm just going to remove...